src/views/run-form.js

```diff
@@ -0,0 +1,41 @@
+export default Backbone.View.extend({
+  tagName: 'form',
+
+  events: {
+    submit(ev) {
+      ev.preventDefault();
+
+      var time = this.$el.find('#time').val();
```

Comment:
Looks like your `inputs` in your template don't have the matching `id` any more. Either update the `id` on the inputs or these selectors.